File: Punto_17_02/Punto.java
package Tareas_121;

public class Punto {
    public float x; float y;
    //constructor
    public Punto(float x, float y){
        this.x = x;
        this.y = y;
    }
    
    //metodos
    public String coord_cartesianas(){
        return "(" + x + "," + y + ")";
    }
    public String coord_polares(){
        double r = Math.sqrt( x * x + y * y);
        double theta = Math.atan2(y, x); //en radianes
        return"(r=" + r + ", 0=" + theta + ")";
    }
    
    //toString
    public String toString(){
        return "Punto en coordenadas cartesianas: " + coord_cartesianas();
    }
}

File: Practica_1/Punto.py
import math 
class Punto:
    def __init__(self, x: float, y: float):
        self.x = x
        self.y = y
# Metodos
def coord_cartesianas(self):
    return f"({self.x}, {self.y})"

def coord_polares(self):
    r = math.sqrt(self.x**2 + self.y**2)
    theta = math.atan2(self.y, self.x) #en radianes
    return f"(r={r}, 0={theta})"

#Metodo toString
def __str__(self):
    returnf"punto en coordenadas cartesianas: {self.coord_cartesianas()}"